# src/utils/exceptions.py


class DataValidationError(Exception):
    """Custom exception for data validation errors."""

    pass


class ConfigError(Exception):
    """Custom exception for configuration errors."""

    pass
